svelte/src/scripts/draw/drag-and-drop.ts

```diff
@@ -54,23 +55,12 @@ export function addDragAndDrop(
 				// Rerender nodes
 				updateNodePosition(node, svgElement);
 
-				// Need to find all relevant links to rerender them.
-				// TODO it might be faster to just rerender every link globally.
-				const links = new Set<GraphDataEdge>();
-				const rec = (nodes: GraphDataNode[]) => {
-					nodes.forEach(n => {
-						[...n.incomingLinks, ...n.outgoingLinks].forEach(l => links.add(l));
-						rec(n.members);
-					});
-				};
-				rec(nodes);
-
-				renderLinks([...links], nodesDictionary, linkCanvas, drawSettings);
+				renderLinks(links, nodesDictionary, linkCanvas, drawSettings);
```

Comment:
I deleted this because I thought it's unnecessary as the renderedLinks object only contains the relevant links